driver/integration_test.go

```diff
@@ -245,7 +248,20 @@ func newDB(t *testing.T, n int) (*sql.DB, []*nodeHelper, func()) {
 	require.NoError(t, store.Set(context.Background(), infos))
 
 	log := logging.Test(t)
-	driver, err := driver.New(store, driver.WithLogFunc(log))
+
+	// all driver options below (except logging) represent the default settings,
+	// and are included for code coverage
```

Comment:
Please revert this to the previous version, i.e. without the various options, unless there is a value that differs from the default, so the test is closer to what users get if they don't use any tweaking.

It's okay if coverage decreases a bit.

If we want to increase coverage and exercise the new knobs, then it'd be better to add ad-hoc tests for them, showing that they actually have the desired effect (e.g. reducing the attempt timeout). However, I don't think it's necessary, so it's up to you if wish or not.

Reply:
My commit lowered the coverage threshold so that a merge wasn't gonna happen. Could I back this out and add another "default knobs" test to make it happy?

Reply:
As said, it's okay if your commit lowers the coverage. I'll merge the branch anyways, there will just a be a "meh" comment from the coverage bot, no big deal :)

Reply:
I missed your last comment and migrated the options test to its one simple sanity check. I hope its not unreasonable. I can remove it if you wish.